tests/flow.rs

```diff
@@ -613,22 +614,136 @@ fn test_blind_sign_request() -> CredxResult<()> {
+    blind_claims_1: &BTreeMap<String, ClaimData>,
+    blind_claims_2: &BTreeMap<String, ClaimData>,
+    nonce: [u8; 16],
+) -> CredxResult<(Presentation<BbsScheme>, PresentationSchema<BbsScheme>)> {
+    // use the same link_secret to value mapping here
+    let (request_1, blinder_1) = BlindCredentialRequest::new(issuer_public_1, blind_claims_1)?;
+    let (request_2, blinder_2) = BlindCredentialRequest::new(issuer_public_2, blind_claims_2)?;
+
+    let blind_bundle_1 = issuer_1.blind_sign_credential(
+        &request_1,
         &btreemap! {
-            "identifier".to_string() => RevocationClaim::from(CRED_ID).into(),
+            "identifier".to_string() => RevocationClaim::from(cred_id).into(),
             "name".to_string() => HashedClaim::from("John Doe").into(),
             "address".to_string() => HashedClaim::from("P Sherman 42 Wallaby Way Sydney").into(),
             "age".to_string() => NumberClaim::from(30303).into(),
         },
     )?;
+    let blind_bundle_2 = issuer_2.blind_sign_credential(
+        &request_2,
+        &btreemap! {
+            "identifier".to_string() => RevocationClaim::from(cred_id).into(),
+            "name".to_string() => HashedClaim::from("Jane Doe").into(),
+            "address".to_string() => HashedClaim::from("Sydney").into(),
+            "age".to_string() => NumberClaim::from(30304).into(),
+        },
+    )?;
+
+    let credential_1 = blind_bundle_1.to_unblinded(blind_claims_1, blinder_1)?;
+    let credential_2 = blind_bundle_2.to_unblinded(blind_claims_2, blinder_2)?;
+
+    let sig_st_1 = SignatureStatement {
+        disclosed: btreeset! {},
+        id: "1".to_string(),
+        issuer: issuer_public_1.clone(),
+    };
+    let sig_st_2 = SignatureStatement {
+        disclosed: btreeset! {},
+        id: "2".to_string(),
+        issuer: issuer_public_2.clone(),
+    };
+
+    let credentials = indexmap! {
+        sig_st_1.id.clone() => credential_1.credential.into(),
+        sig_st_2.id.clone() => credential_2.credential.into()
+    };
+
+    let pres_sch_id = random_string(16, rand::thread_rng());
 
-    let _ = blind_bundle.to_unblinded(&blind_claims, blinder)?;
+    let eq_st = EqualityStatement {
+        id: random_string(16, rand::thread_rng()),
+        ref_id_claim_index: indexmap! {
+            sig_st_1.id.clone() => 1,
+            sig_st_2.id.clone() => 1,
+        },
+    };
+    let pres_sch_1 = PresentationSchema::new_with_id(
+        &[
+            sig_st_1.clone().into(),
+            sig_st_2.clone().into(),
+            eq_st.clone().into(),
+        ],
+        &pres_sch_id,
+    );
+
+    let presentation = Presentation::create(&credentials, &pres_sch_1, &nonce)?;
+
+    // Prover and verifier each use PresentationSchema generated independently.
+    // 'pres_sch_2' to be used by verifier.
+    let pres_sch_2 = PresentationSchema::new_with_id(
+        &[sig_st_1.into(), sig_st_2.into(), eq_st.into()],
+        &pres_sch_id,
+    );
```

Comment:
Did you intend to make two presentations? It looks like you are just comparing schemas and not presentations. The proofs shouldn't be exactly equal in value but essentially prove the same thing.

Reply:
This test extension is not comparing schema nor presentations.

It creates the 2nd presentation schema to be used by the verifier IFF `Presentation::create` succeeds.  See the comment about the creation of the 2nd schema.

Reply:
@mikelodder7 I responded to your question.  Is this PR mergable now?